src/createNumbersGenerator.js

```diff
@@ -25,8 +25,19 @@
  *
  * @return {number}
  */
-function createNumbersGenerator(numbers, min, max) {
-  // write code here
+function createNumbersGenerator(numbers = '', min = -Infinity, max = Infinity) {
+  let index = 0;
+  let numbersCopy = numbers;
```

Comment:
you can't create copy of array or object in this way.
in this way, you copy the only link to array but not the values from an array. If you change `numbersCopy`, also you change `numbers `